identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-with-verification.jsp

```diff
@@ -891,53 +892,55 @@
         var consecutiveChr = /([^])\1+/g;
         var errorMessage = getErrorMessage();
 
-        if (passwordConfig.minLength> 0 || passwordConfig.maxLength > 0) {
-            document.getElementById("length").innerHTML = '<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "must.be.between")%>' +
-                " " + (passwordConfig.minLength ?? 8) +
-                " " + '<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "and")%>' +
-                    " " + (passwordConfig.maxLength ?? 30) + " " +
-            '<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "characters")%>';
-            $("#length-block").css("display", "block");
-        }
-        if (passwordConfig.minNumber > 0) {
-            document.getElementById("number").innerHTML = '<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "at.least")%>'
-                + " " + passwordConfig.minNumber + " "
-                + '<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "numbers")%>';
-            $("#number-block").css("display", "block");
-        }
-        if ((passwordConfig.minUpperCase > 0) || passwordConfig.minLowerCase > 0) {
-            let cases = [];
-            if (passwordConfig.minUpperCase > 0) {
-                cases.push(passwordConfig.minUpperCase + " "
-                    + '<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "uppercase")%>');
+        if (passwordValidationBlock.length > 0) {
```

Comment:
Won't this throw a runtime error such as `Cannot read length of undefined`, if the passwordValidationBlock is not found?

Reply:
Internally, jQuery handles this process. If there is no matching element, jQuery will return an empty object. To check the existence of an element, it is recommended to use the condition `if (passwordValidationBlock.length > 0)`.

Reply:
Ah nice